modules/global/src/com/haulmont/cuba/core/global/GlobalConfig.java

```diff
@@ -249,4 +249,14 @@
     @Property("cuba.deepCopyNonPersistentReferences")
     @DefaultBoolean(true)
     boolean getDeepCopyNonPersistentReferences();
+
+    /**
+     * Disable ESCAPE in queries with LIKE for specified data stores.
+     */
+    @Property("cuba.disableEscapingLikeForDataStores")
+    @Source(type = SourceType.DATABASE)
+    @Factory(factory = CommaSeparatedStringListTypeFactory.class)
+    @Stringify(stringify = CommaSeparatedStringListStringify.class)
+    List<String> getDisableEscapingLikeForDataStores();
+    void setDisableEscapingLikeForDataStores(List<String> names);
```

Comment:
Setter is not needed if you are not going to change the property programmatically. App properties screen doesn't need it.